CONFIG_REFERENCE.md

```diff
@@ -194,25 +194,50 @@ specify them explicitly!)
   - `src_hostname`
     Hostname reported by the client in the EHLO/HELO command.
 
-  See below for example.
+    Example:
+    ```
+    match rcpt "/@emersion.fr$/" {
```

Comment:
Ugh. I'm confused.

Reply:
Right, I should replace it with rcpt_domain, forgot to do that when moving examples around.